spec/controllers/tag_wranglers_controller_spec.rb

```diff
@@ -17,4 +17,61 @@
       end
     end
   end
+
+  describe "#report_csv" do
+    let!(:tag1) { create(:freeform, last_wrangler: user, unwrangleable: true) }
+    let!(:tag2) { create(:relationship, last_wrangler: user) }
+
+    context "when anonymous" do
```

Comment:
fwiw, we'd usually say something more like "when logged out" here, since "anonymous" has a special meaning on the Archive.